api/app/Models/Timezones.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace App\Models;
+
+use Illuminate\Support\Collection;
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * Class Timezones
+ * @package App\Models
+ *
+ * Note this model does not have an associated database table as it is an abstract
+ * data model with generated token data.
+ */
+class Timezones extends Model
```

Comment:
Yea I implemented this as a model only so I could do a simple call from the documentation (to use a model factory) to explain the requirements. I'm happy for it to be a service/factory, but bear in mind that I could break the `ModelFactory` call in the docs.